server/models/List.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose')
+
+const ListSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true,
+  },
+  image: {
+    type: String,
+  },
+  products: [{ type: mongoose.Schema.Types.ObjectId, ref: 'Product' }],
+  userId: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'User',
+  },
```

Comment:
Optional, but when deciding how to establish a relationship between two models it's helpful to think about how you will query the database and how that will scale. At the moment, if you want to fetch all the lists for a user you would have to find lists by userId. That's fine, but if you did the opposite and stored these as an array on the user, you could just populate that array when you get the user by id.